packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -3,10 +3,12 @@
 
 :local {
   .nav-container {
+    background: inherit;
```

Comment:
i don't think this is having the effect you are looking for completely, profile shouldn't stand out with its own background
<img width="393" alt="screen shot 2017-08-14 at 9 48 38 am" src="https://user-images.githubusercontent.com/30411845/29274429-dff5dec0-80d5-11e7-8ae7-900958c4c926.png">
: